packages/trader/src/AppV2/Components/Guide/Description/ContractDescription/touch-no-touch-trade-description.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { CONTRACT_LIST, getContractDescription } from 'AppV2/Utils/trade-types-utils';
+
+const TouchNoTouchTradeDescription = () => {
+    const [touch, no_touch] = CONTRACT_LIST.TOUCH_NO_TOUCH.split('/');
+    const content = [
+        { type: 'heading', text: <Localize i18n_default_text='Touch' /> },
+        {
+            type: 'paragraph',
+            text: (
+                <Localize
+                    i18n_default_text='If you select “<0>Touch</0>”, you win the payout if the market touches the barrier at any time during the contract period.'
+                    components={[<span className='description__content--bold' key={0} />]}
+                />
+            ),
+        },
+        {
+            type: 'video',
+            text: touch,
+        },
+        { type: 'heading', text: <Localize i18n_default_text='No Touch' /> },
+        {
+            type: 'paragraph',
+            text: (
+                <Localize
+                    i18n_default_text='If you select “<0>No Touch</0>”, you win the payout if the market never touches the barrier at any time during the contract period.'
+                    components={[<span className='description__content--bold' key={0} />]}
+                />
+            ),
+        },
+        {
+            type: 'video',
+            text: no_touch.split(' ').join('_'),
```

Comment:
Hmm, why do you need to import text and do all these text operations? Why not type the required text here?

Reply:
Yep) Sometimes I'm going crazy with constants, sorry)))
Will fix here and it the similar places